README.adoc

```diff
@@ -66,59 +66,59 @@ See also: <<your-profile-disturbs-me>>.
+* Dictatorships are more likely to start <<war,wars>> or other crazy policies like the Great Leap Forward, which completely destroy the economy in one go
 * society becomes richer when people know that they can do their startups, get rich, and stay in the country without fear of being persecuted unfairly and losing everything instead of migrating to Canada.
 +
-Any criticism of the government, even if constructive, is taken as menace to power, and more likely to be shut down, which makes the government and just becomes less efficient since there is less feedback.
-* governments are monopolies, and the more powerful they are, the worst it is for competition an efficiency in general. E.g.: the startup with better government ties wins, instead of the most efficient one.
+Any criticism of the government, even if constructive, is taken as a menace to power, and more likely to be shut down, which makes the government and just becomes less efficient since there is less feedback.
+* Governments are monopolies, and the more powerful they are, the worst it is for competition an efficiency in general. E.g., the startup with better government ties wins, instead of the most efficient one.
 * dictatorships need <<censorship>> to survive, and they must control all information to make themselves always look good.
 +
 As a result, knowledge of problems flows more slowly, and therefore they also take longer to solve.
 +
-Maybe this hurts my argument, but Hillary agrees: https://www.youtube.com/watch?v=ccGzOJHE1rw&feature=youtu.be&t=2110 "Secretary Clinton Speaks on Internet Freedom", U.S. Department of State, 2010-01-22 :-)
+Maybe this hurts my argument, but Hillary agrees: https://www.youtube.com/watch?v=ccGzOJHE1rw&feature=youtu.be&t=2110 "Secretary Clinton Speaks on Internet Freedom," U.S. Department of State, 2010-01-22 :-)
 +
 ____
-But countries that restrict free access to information or violate the basic rights of internet users risk walling themselves off from the progress of the next century
+But countries that restrict free access to information or violate the basic rights of internet users risk walling themselves off from the progress of the next century.
 ____
 +
-Also mentioned at: link:https://youtu.be/d3dE_LDz_9E?t=1681[Google versus China - VPRO documentary - 2011]
+Also mentioned at link:https://youtu.be/d3dE_LDz_9E?t=1681[Google versus China - VPRO documentary - 2011]
 +
-This is also well illustrated in the link:https://en.wikipedia.org/wiki/Chernobyl_(miniseries)[HBO 2019 miniseries "Chernobyl"], which suggests that part of the reason why Chernobyl happened is because of the Soviet Union's obsession to save face.
+This is also well illustrated in the link:https://en.wikipedia.org/wiki/Chernobyl_(miniseries)[HBO 2019 miniseries "Chernobyl,"] which suggests that part of the reason why Chernobyl happened is because of the Soviet Union's obsession to save face.
```

Comment:
I think this one was correct?

Reply:
OK, Americans are crazy: https://english.stackexchange.com/questions/48447/placement-of-comma-before-quotation-mark